projects/storefrontapp-e2e-cypress/cypress/plugins/index.js

```diff
@@ -15,13 +14,17 @@ module.exports = (on, config) => {
 
     deleteFolder(folderName) {
       return new Promise((resolve, reject) => {
-        rmdir(folderName, { maxRetries: 3, recursive: true }, (err) => {
-          if (err) {
-            console.error(err);
-            return reject(err);
+        fs.rm(
+          folderName,
+          { maxRetries: 3, recursive: true, force: true },
```

Comment:
what is the purpose of `force: true`?

Reply:
from guide https://nodejs.org/api/fs.html#fsrmpath-options-callback
When true, exceptions will be ignored if path does not exist. Default: false.